src/store/pageview.ts

```diff
@@ -3,25 +3,25 @@ import { PageView, StorageKeys, PageFeatureResult } from '../../types';
 
 const DEFAULT_MAX_FEATURES_SIZE = 300;
 
-class ViewStore {
-  pageViews: PageView[];
-  storageSize: number;
+export class ViewStore {
```

Comment:
The stores should not be exported - it is important they are singletons to avoid overwrite of data.